### PR TITLE
feat: implement first pld generation wrapped in app structure

### PR DESCRIPTION
**Pull request by mindoodoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,59 @@
...
+
+| **Mandatory**      | **Key**             | **Description**                                      |
+| ------------------ | ------------------- | ---------------------------------------------------- |
+| :white_check_mark: | github_api_key      | Fine grained token                                   |
+| :white_check_mark: | project_number      | Github projects number                               |
+| :white_check_mark: | lucid_client_id     | Lucid OAuth client ID                                |
+| :white_check_mark: | lucid_client_secret | Lucid OAuth client secret                            |
+| :white_check_mark: | lucid_access_token  | Lucid access token generated through OAuth2 process  |
+| :white_check_mark: | lucid_refresh_token | Lucid refresh token generated through OAuth2 process |
+| :white_check_mark: | document_id         | Lucid document id                                    |
```
Comment (nydragon) [inline README.md:33]: replace `:white_check_mark:` by the Unicode character ✅ so the character is correctly rendered  when in text mode.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,59 @@
...
-- [X] Graphql request to get cards
-  - [ ] Handle paging
-- [ ] Cards parsing
-- [ ] Markdown generating
+Optional flags :
 
-## Improvements
+-c, --config
+  Specify config file path.
+```
```
Comment (nydragon) [inline README.md]: I do not think it makes sense to specify the output of clap's help, can get out of sync easily and people will never refer to the README to check for flags.

Much easier to just check via binary

File: README.md (hunk 1 above)
Comment (mindoodoo) [reply README.md:33]: Unfortunately does not render for me so I'll keep `:white_check_mark:` for the time being.

Comment (nydragon) [reply README.md:33]: That is a bad reason

**Pull request merged**